File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/changelly/GetCurrenciesResponseBean.java
package com.androidevlinux.percy.UTXO.data.models.changelly;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by percy on 15/11/2017.
 */

public class GetCurrenciesResponseBean {

    @SerializedName("jsonrpc")
    @Expose
    private String jsonrpc;
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("result")
    @Expose
    private List<String> result = null;


    public String getJsonrpc() {
        return jsonrpc;
    }

    public void setJsonrpc(String jsonrpc) {
        this.jsonrpc = jsonrpc;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public List<String> getResult() {
        return result;
    }

    public void setResult(List<String> result) {
        this.result = result;
    }
}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/changelly/GetCurrenciesResponseBean.kt
package com.androidevlinux.percy.UTXO.data.models.changelly

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by percy on 15/11/2017.
 */

class GetCurrenciesResponseBean {

    @SerializedName("jsonrpc")
    @Expose
    var jsonrpc: String? = null
    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("result")
    @Expose
    var result: List<String>? = null
}
